dbsnap_verify/__init__.py

```diff
@@ -0,0 +1,154 @@
+from time import time
+
+import datetime
+
+import json
+
+import boto3
+s3 = boto3.client("s3")
+
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
Maybe make this configurable via environment variable?  You'll also need to setup a basic logger.

Reply:
Though the basic logger may already be setup in lambda for you - not sure.